source/index.html.md

```diff
@@ -203,7 +203,7 @@ This endpoint loads customers by csv files
 
 The CSV file must have the following records, and must have a heading matching the following.
 
-**First name;Last name;Phone number;Customer number**
+**FIRST NAME;LAST NAME;PHONE NUMBER;CUSTOMER NUMBER**
```

Comment:
I didn't ask you to capitalize it. Its `First name`, not `FIRST NAME`. Pay attention. It should be as i sent you.

Reply:
Please this is what you said in the comment "Followed by the following in caps on a separate line." I was even confused